deployments/container/Dockerfile

```diff
@@ -102,11 +102,12 @@ LABEL org.opencontainers.image.source="https://github.com/NVIDIA/k8s-dra-driver-
 # https://docs.docker.com/build/building/multi-platform/#install-qemu-manually
 RUN mkdir /licenses && mv /NGC-DL-CONTAINER-LICENSE /licenses/NGC-DL-CONTAINER-LICENSE
 
+COPY /hack/kubelet-plugin-prestart.sh /usr/bin/kubelet-plugin-prestart.sh
+COPY /templates /templates
+
```

Comment:
Can we do this last -- also if we are separating them into "groups", then there should be a newline after copy from the toolkit stage, before the copy from the build stage. Otherwise, just smash them all together.